### PR TITLE
[Featurev]Added 404 error page.

### PR DESCRIPTION
**Pull request by Deepanshu0703**
## Related Issue

It is a 404 error page

Closes: #172

## Description of Changes

No change in the existing code just added few things

 -modified:   public/assets/css/style.css
 -Added :      components/Error.js
 -Added  :       pages/404.page.js
 -Added   :    public/assets/img/hero/download.png

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| ![Original screenshot]
image | ![Updated screenshot]image|

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Comment by anmode** [comment]
You can keep white background... and optimize it

**Comment by anmode** [comment]
we are using this color `#845ec2`. make sure to setup theme button color and on click just make it little dark.

**Comment by Deepanshu0703** [comment]
i have used this image 
image
 and the background is this 

image

and used the same button color that home page is with same hovering effect so do i have to change the color

**Comment by anmode** [comment]
please do not that image.. not look great for 404 page. tagging @AmanJain18 for more details.

**Comment by Deepanshu0703** [comment]
I provided some example where he selected this so i implemented this one
Still if there is any change required I will do it . So please suggest the change @AmanJain18

**Comment by anmode** [comment]
@Deepanshu0703 could you look at design.. a simple white shade (Not proper white) background with nice vector works

**Comment by Deepanshu0703** [comment]
I saw this animation may i use this one @anmode 

Screenshot (28)

**Comment by Deepanshu0703** [comment]
Or this one , this is also an animation.

image

**Comment by anmode** [comment]
Think in the terms of Grabtern:
Idea - Connecting mentor and mentee ( 1-1 On mentorship)
so how would like 404 page would look like
@AmanJain18 @MohitGupta121 @Abhinavcode13  any ideas you wanna provide? or any suggestions

**Comment by Abhinavcode13** [comment]
image
somewhat like this @anmode

**Comment by anmode** [comment]
> image somewhat like this @anmode

yeah! It's cool @Abhinavcode13

**Comment by Deepanshu0703** [comment]
Updated with the new image in the left and some description and back to homepage option in the right side

**Comment by MohitGupta121** [comment]
> > image somewhat like this @anmode
> 
> yeah! It's cool @Abhinavcode13

@anmode This looks good to me.

**Comment by AmanJain18** [comment]
ig this will be good ? @anmode

**Comment by anmode** [comment]
>  ig this will be good ? @anmode

unstop pr acha lagega....unki site ke hisab. see ek baarr

**Comment by Deepanshu0703** [comment]
I have updated the error page with this image  in the left and back to home option in left 

image

**Review by anmode — APPROVED** [review]
Our theme color new is `#845ec2`
Remember background should not be pure white* See the new theme
https://doctor-ruby.vercel.app/. ( keep the light mode)
and 
keep the css file in style folder with name 404.css 
Btw it looks cool!

**Comment by Deepanshu0703** [comment]
here is the screenshot

image

**Review by MohitGupta121 — APPROVED** [review]
@Deepanshu0703 screenshot LGTM, Thanks.

File: public/assets/css/style.css [hunk 1]
```diff
@@ -5999,3 +6006,66 @@ del {
     gap: 80px !important;
   }
 }
+
+/*------------------
+  Error Page
+--------------------*/
+.error {
+  display: flex;
```
Comment (anmode) [inline public/assets/css/style.css:6014]: Move the css file to `styles/404.css`

**Review by anmode — CHANGES_REQUESTED** [review]
1. png should in webp
2. Please Use our theme color in button and try to see if you can modify image to in that color
3. why 404 not coming when i does `grabtern\ddfjd` but when i did `grabtern\sdf\dfd` it was fine

**Comment by anmode** [comment]
@Deepanshu0703 ?

**Pull request closed without merging**